lib/config.py

```diff
@@ -263,9 +264,11 @@ def parse(self):
 
         # update iso node with iso subcommand args and then drop them from args
         for key, value in args.items():
-            if key in config['iso']:
-                config['iso'][key] = value
-                args.pop(key)
+            for node in COMMAND_TO_CONFIG_NODE.values():
```

Comment:
It seems to me both are functionally equivalent. I guess your point is making code shorter. Done